spec/vcr/ShipmentsController/GET_shipment_/is_successful.yml

```diff
@@ -0,0 +1,176 @@
+---
+http_interactions:
+- request:
+    method: post
+    uri: https://onlinetools.ups.com/ups.app/xml/Rate
+    body:
+      encoding: UTF-8
+      string: |
+        <?xml version="1.0"?>
+        <AccessRequest>
+          <AccessLicenseNumber>7CE85DED4C9D07AB</AccessLicenseNumber>
```

Comment:
You should add VCR responses to your `.gitignore`, because they can expose secrets. In this case, I can see your access license number. In a production setting, that could cost the company a lot of money. In general, I think it's a good idea to hide anything returned by VCR, webmock, and other HTTP stubbing tools, because they may expose secrets.